test/Hpack/ConfigSpec.hs

```diff
@@ -1151,3 +1151,25 @@ spec = do
         it "returns an error" $ \dir -> do
           let file = dir </> "package.yaml"
           readPackageConfig file `shouldReturn` Left [i|#{file}: Yaml file not found: #{file}|]
+
+      context "when included yaml file does not exist" $ do
+        it "returns an error" $ \dir -> do
+          let file = dir </> "package.yaml"
+          let included = "authors.yaml"
+          let includedAbs = dir </> included
+          writeFile file [i|
+            author: !include "#{included}"
+            |]
+          readPackageConfig file `shouldReturn` Left [i|#{file}: Yaml file not found: #{includedAbs}|]
+
+    it "accepts !include directive" $ do
+      let included = "authors.yaml"
+      withPackageConfig [i|
+        author: !include "#{included}"
+        |]
+        (writeFile included [i|
+          - A1
+          - A2
+        |]
+        )
+        (packageAuthor >>> (`shouldBe` ["A1", "A2"]))
```

Comment:
Oh, I considered it part of "hpack specification" and naturally wrote a hspec test for that. :-)